temporalio/worker/_workflow_instance.py

```diff
@@ -422,8 +422,6 @@ def activate(
                 continue
             i += 1
 
-        if seen_completion:
-            self._warn_if_unfinished_handlers()
```

Comment:
Was this removed intentionally?

Reply:
Yeah, sorry I pushed a temp commit for the cancellation sample I sent you earlier, to force push over later. Force-pushed now.